sdk/eventhub/azure-eventhubs/tests/asynctests/test_longrunning_send_async.py

```diff
@@ -56,26 +52,16 @@ def check_send_successful(outcome, condition):
     deadline = time.time() + duration
     total = 0
 
-    def data_generator():
-        for i in range(args.batch):
-            yield b"D" * args.payload
-
-    if args.batch > 1:
-        logger.info("{}: Sending batched messages".format(pid))
-    else:
-        logger.info("{}: Sending single messages".format(pid))
-
```

Comment:
Why was this removed?
Without message batching this performance test will never get the needed throughput.
Batched messages should be the default operation of this test